### PR TITLE
!str #19137: Rename inlet and outlet to in and out on Flow/Source/SinkShape

### PR DESCRIPTION
**Pull request by drewhk**
refs #19137

### CODE REVIEW

**Comment by ktoso** [comment]
LGTM, thanks Endre

File: akka-bench-jmh-dev/src/main/scala/akka/stream/MaterializationBenchmark.scala [hunk 1]
```diff
@@ -58,7 +58,7 @@ object MaterializationBenchmark {
     RunnableGraph.fromGraph(GraphDSL.create(Source.single(())) { implicit b ⇒ source ⇒
       import GraphDSL.Implicits._
       val flow = Flow[Unit].map(identity)
-      var outlet: Outlet[Unit] = source.outlet
+      var outlet: Outlet[Unit] = source.out
```
Comment (hepin1989) [inline akka-bench-jmh-dev/src/main/scala/akka/stream/MaterializationBenchmark.scala]: will this be rename to out too?

Comment (drewhk) [inline akka-bench-jmh-dev/src/main/scala/akka/stream/MaterializationBenchmark.scala]: ok (that is an internal one though).

**Comment by rkuhn** [comment]
:+1:

**Comment by ktoso** [comment]
Needs a rebase @drewhk

**Comment by drewhk** [comment]
Rebased. Merging after validation.

**Pull request merged**